src/components/tests/Profile.test.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { MemoryRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import { render } from '@testing-library/react';
+import Profile from '../../components/Profile/Profile'
+import store from '../../Redux/configureStore';
+
+describe('Jest Snapshot testing suite', () => {
+  it('Matches DOM Snapshot', () => {
+    const { container } = render(
+      <MemoryRouter>
+        <Provider store={store}>
+          <Profile />
+        </Provider>
+      </MemoryRouter>,
+    );
+    expect(container.firstChild).toMatchSnapshot();
+  });
+});
```

Comment:
- Some of your tests are failing, kindly fix them 👍🏽
![Screenshot from 2022-08-27 01-50-23](https://user-images.githubusercontent.com/44624138/187008223-95a995dc-6d8a-4d8a-9e9a-925ff5f21311.png)
![Screenshot from 2022-08-27 01-50-58](https://user-images.githubusercontent.com/44624138/187008218-86c2c986-0074-49e9-9c0a-ce76a7a4dcc1.png)
![Screenshot from 2022-08-27 01-51-21](https://user-images.githubusercontent.com/44624138/187008224-98e1be52-1d81-41fd-9a21-f0a93ac7efcc.png)

Reply:
This change has been done well